contributors/design-proposals/daemonset-update.md

```diff
@@ -206,6 +244,18 @@ Users can use `kubectl rollout` to monitor DaemonSet updates:
 
 - `kubectl rollout status daemonset/<DaemonSet-Name>`: to see the DaemonSet
   upgrade status 
+- `kubectl rollout history daemonset/<DaemonSet-Name>`: to view the history of
+  DaemonSet updates. We use `PodTemplate` created by DaemonSets to store update
+  history.
+- `kubectl rollout undo daemonset/<DaemonSet-Name>`: to rollback a DaemonSet
+
+#### API
+
+Implement a subresource for DaemonSet history (`daemonsets/foo/history`) that
```

Comment:
Is this generic (pod template list, intended for generic clients) or type specific (array of daemonsets, with only some fields preserved)?